mlir/include/mlir/Conversion/ArithToAMDGPU/ArithToAMDGPU.h

```diff
@@ -20,7 +20,8 @@ class Pass;
 #include "mlir/Conversion/Passes.h.inc"
 
 namespace arith {
-void populateArithToAMDGPUConversionPatterns(RewritePatternSet &patterns);
+void populateArithToAMDGPUConversionPatterns(RewritePatternSet &patterns,
+                                             bool saturateFP8Truncf);
```

Comment:
Could you add a documentation comment?
also nit:
```suggestion
void populateArithToAMDGPUConversionPatterns(RewritePatternSet &patterns,
                                             bool saturateFP8TruncF);
```